### PR TITLE
Create PR template

### PR DESCRIPTION
**Pull request by ckingbailey**
addresses 1/5 of #24 

Adds a pull request template in the /.github folder, which Github should use for all PRs

### Github screenshot
Screen Shot 2019-07-30 at 7 48 52 PM

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+##### _You may delete all the italicized instructional headings_
+##### _Please indicate which issue your Pull Request address_
```
Comment (anlawyer) [inline .github/pull_request_template.md]: Should be “addresses”?

Comment (ckingbailey) [reply .github/pull_request_template.md]: Found this list of keywords that will make a PR close an issue https://help.github.com/en/articles/closing-issues-using-keywords

Comment (ckingbailey) [reply .github/pull_request_template.md]: variations of "closes", "fixes", "resolves" should all work

Comment (anlawyer) [reply .github/pull_request_template.md]: Yep! I just mean the sentence “please indicate which issue your Pull Request _addresses_”. Am I reading it wrong?

Comment (ckingbailey) [reply .github/pull_request_template.md]: you're so right

**Pull request merged**